North-Seattle-College/ad440-winter2021-tuesday-repo
# Sprint2-Team1-Task3

## Changes
- Moved GetTasks logic from dbHandler to __init__.py
- Setup basic debugging logging

## Steps to test
- Fetch branch
- Open 'functions' folder in VS code
- Within VS code open terminal and input `. .venv/Scripts/Activate`
  - Confirm that you are now in the virtual environment
  - input `pip list` (This should display the required packages "azure functions, pip, pyodbc")
- Press F5 to boot the azure function locally
- Go to the getTasks url to test

This is related to issue #54
